### ISSUE TITLE
Feature request: support for x-discriminator

### ISSUE THREAD

**Issue by RomanGotsiy** [Mermade/oas-kit#70]
`swagger2openapi` converts `x-required`, `x-nullable` and a few others vendor extensions to corresponding OpenAPI 3 features.

Would be great to also support `x-discriminator`:

```yaml
x-discriminator:
  propertyName: type
  mapping:
    visa: "#/definitions/paymentMethodVisa"
    mastercard: "#/definitions/paymentMethodMastercard"
    amex: "#/definitions/paymentMethodAmex"
```

One of the things to not forget is to map JSON pointers from `mapping` field to new locations.

**Comment by MikeRalphson** [comment]
@RomanGotsiy will be included in the next release.

**Comment by leonardehrenfried** [comment]
Hello @MikeRalphson, thanks for this great library! 

I'm eagerly awaiting the newest release of this library. When have you planned on doing the release?

(Sorry to be a pest!)

**Comment by MikeRalphson** [comment]
HI @leonardehrenfried - thanks, and you're not being a pest! I'm kind of waiting on [lerna 3](https://github.com/lerna/lerna) to come out of beta, to try and ensure there are no problems publishing from a new mono-repo layout. If it drags on too long, I'll bite the bullet and attempt to publish the new version.

**Comment by MikeRalphson** [comment]
Included in `swagger2openapi` `v3.0.x`.
